pyccl/nl_pt/tracers.py

```diff
@@ -79,6 +61,8 @@ class PTTracer(object):
     in a perturbation theory framework to provide N-point
     correlations.
     """
+    __repr_attrs__ = ('type', 'biases')
```

Comment:
nitpicking but it is recommended to end tuples with a comma, just in case it only has one element

Reply:
Until it does, in which case you'll get a very weird error. Nevermind.